### PR TITLE
convert malaria consortium pillow checkpoint to use sql

### PR DESCRIPTION
**Pull request by czue**
depends on https://github.com/dimagi/pillowtop/pull/178

:blowfish: @snopoke cc @calellowitz 

if this goes well I'll do all the fluff pillows in a few days

### CODE REVIEW

File: corehq/apps/cleanup/pillow_migrations.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        new_checkpoint = DjangoPillowCheckpoint(
+            checkpoint_id=pillow_config.get_instance().checkpoint.checkpoint_id,
+            sequence=legacy_checkpoint['seq'],
+            old_sequence=legacy_checkpoint.get('old_seq', None)
+        )
+        new_checkpoint.save()
+    except Exception as e:
+        logging.exception('Failed to update pillow checkpoint. {}'.format(e))
+
+
```
Comment (dimagimon) [inline corehq/apps/cleanup/pillow_migrations.py]: W391 blank line at end of file

File: custom/reports/mc/models.py [hunk 2]
```diff
@@ -5,6 +5,8 @@
 from corehq.fluff.calculators import xform as xcalculators
 from fluff.filters import NOTFilter, ANDFilter, ORFilter
 from fluff.models import SimpleCalculator
+from pillowtop.checkpoints.manager import get_django_checkpoint_store, \
```
Comment (dimagimon) [inline custom/reports/mc/models.py]: F401 'get_django_checkpoint_store' imported but unused

**Comment by snopoke** [comment]
looks good. Wait for tests

**Comment by czue** [comment]
good call - just fixed a test

**Pull request merged**